### PR TITLE
Fix bug appeared in vizarr

### PR DESCRIPTION
**Pull request by oeway**
This is a fix to the bug that causes vizarr to fail

@manzt

### CODE REVIEW

File: javascript/src/rpc.js [hunk 1]
```diff
@@ -766,11 +766,9 @@ export class RPC extends MessageEmitter {
         if (aObject._rintf) {
           const temp = aObject._rtype;
           delete aObject._rtype;
-          aObject = await this._decode(aObject, withPromise);
-          aObject._rtype = temp;
-        }
-        delete aObject._rintf;
```
Comment (oeway) [inline javascript/src/rpc.js:772]: This was the line that causes vizarr to fail.

**Comment by manzt** [comment]
Great, thank you so much! Is there something I will need to change to get the examples working again?

It would be ideal to have our examples versioned in a way that are future proof as well.

**Comment by oeway** [comment]
@manzt Welcome! No, we don't need to change anything. 

This is how the dependency flows: `imjoy-rpc ==> imjoy-core ==> imjoy-jupyter-extension ==> vizarr`

For ` imjoy-jupyter-extension<=0.2.13` we always use the latest version of imjoy-core, and that cause the issue of vizarr. Since now we pin the version of imjoy-core in imjoy-jupyter-extension, it should be future proof.

Therefore, I suggest in vizarr, we do ` imjoy-jupyter-extension>=0.2.14` to make it future proof.

**Comment by manzt** [comment]
Great I have an open PR here: https://github.com/hms-dbmi/vizarr/pull/65

What about colab, is there something needed for that example?

I can't seem to get this notebook to work:

https://colab.research.google.com/github/hms-dbmi/vizarr/blob/version-imjoy-jupyter-plugin/example/mandelbrot.ipynb#scrollTo=0nilX8jAjphg

**Comment by oeway** [comment]
@manzt Oh, thanks for the reminder, I actually forgot about colab. It was using the latest version of imjoy-core. It should have been fixed for the latest version, but the cdn (jsdelivr.com) requires a few hours to refresh.
I have just merged:  https://github.com/imjoy-team/imjoy-rpc/pull/71 to pin also the imjoy-core version for colab, so we won't have this type of caching issue. Just tried the colab example and it works for me now.

**Pull request merged**